modules/ossm-cr-pilot.adoc

```diff
@@ -5,7 +5,7 @@
 [id="ossm-cr-pilot_{context}"]
 = Istio Pilot example
```

Comment:
All these sections are called "example", but they also contain tables describing the parameters, so this should be called more like "Istio Pilot configuration".

Reply:
Fixed.